docs/docs/deploy/deploy-rasa-plus.mdx

```diff
@@ -78,38 +80,36 @@ Step c: Remove the file on disk, for example:
 
 For the [Rasa Helm Chart](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa), update as follows:
 
-```yaml
-registry: europe-west3-docker.pkg.dev/rasa-releases/rasa-plus/rasa-plus
+<pre><code parentName="pre" className="language-yaml">
+{`registry: europe-west3-docker.pkg.dev/rasa-releases/rasa-plus/rasa-plus
 command:
   - rasa
 image:
   name: rasa-plus
-  tag: 0.1.1
+  tag: ${variables.release}
```

Comment:
this is accurate only if we release `rasa-plus` in lockstep with `rasa`. At this stage that's what we want to do, but we do not have a mechanism to enforce it (soon!)